src/main/scala/example/CborAkkaSerializer.scala

```diff
@@ -0,0 +1,39 @@
+package example
+
+import akka.serialization.Serializer
+import io.bullet.borer.{Cbor, Codec, Decoder, Encoder}
+
+import scala.reflect.ClassTag
+
+trait CborAkkaSerializer[Ser] extends Serializer {
+
+  private var registrations: List[(Class[_], Codec[_])] = Nil
```

Comment:
Hmm it looks to me it's not thread-safe... probably this should be a synchronized collection

Reply:
Or alternatively, a `var` but pointing to sth like an `AtomicReference`